code/datums/components/jump.dm

```diff
@@ -80,6 +79,17 @@
 		to_chat(jumper, span_warning("Catch your breath!"))
 		return
 
+	do_jump(jumper)
+	//Forces all who ride to jump alongside the jumper.
+	for(var/mob/buckled_mob AS in jumper.buckled_mobs)
+		do_jump(buckled_mob, FALSE)
+
+///Performs the jump
+/datum/component/jump/proc/do_jump(mob/living/jumper, use_stamina = TRUE)
+
+	if(TIMER_COOLDOWN_CHECK(jumper, JUMP_COMPONENT_COOLDOWN))
+		return
```

Comment:
keep this in start_jump. The piggybacker should be forced to jump regardless of their own CD.

Reply:
eah